chroma-manager/tests/framework/unit/jenkins_steps/main

```diff
@@ -27,48 +27,6 @@ cd $CHROMA_DIR/chroma-manager
 make requirements
 pip install -r requirements.txt
 
```

Comment:
did you mention this removal of code in the issue comments? apologies if you did and I missed it